source/_components/sensor.sensehat.markdown

```diff
@@ -34,5 +35,119 @@ Configuration variables:
   - 'temperature'
   - 'humidity'
   - 'pressure'
+  is_hat_attached (Optional): True|False boolean; True declares the SenseHAT _is_ physically on the Raspberry Pi
+  
 
+#### Customizing the Sense HAT data
```

Comment:
If you compare the original PR #2009 you will see that the text is different to what is on [/sensor.sensehat](https://home-assistant.io/components/sensor.sensehat/) So you will be able to judge for yourself that the merger failed somehow.
So this PR is essentially the same as #2009 with the addition of the new flag, and description.